core/templates/dev/head/domain/feedback_message/FeedbackMessageSummaryObjectFactory.ts

```diff
@@ -58,22 +57,20 @@ export class FeedbackMessageSummaryObjectFactory {
       newMessageId, newMessageText, null, null, null, null, authorUsername,
       authorPictureDataUrl, new Date());
   }
-  // TODO(#7176): Replace 'any' with the exact type. This has been kept as
-  // 'any' because 'feedbackMessageSummaryBackendDict' is a dict with
-  // underscore_cased keys which give tslint errors against underscore_casing
-  // in favor of camelCasing.
+
   createFromBackendDict(
-      feedbackMessageSummaryBackendDict: any): FeedbackMessageSummary {
+      feedbackMessageSummaryBackendDict:
+      FeedbackMessageSummary): FeedbackMessageSummary {
     return new FeedbackMessageSummary(
-      feedbackMessageSummaryBackendDict.message_id,
+      feedbackMessageSummaryBackendDict.messageId,
```

Comment:
@milekag01, Have you tested these changes (manually)? I think the backend dict will have snake cases.

Reply:
I have not tested it manually. I am facing some problems in understanding how the services are being used in the components and how are they linked to the backend. Can you please help me here.

Reply:
@milekag01 This domain object is used in the suggestion thread. You can create suggestion for an exploration (From a/c 1) then try adding feedback to the suggestion (From a/c 2), then check the feedback thread through the creator dashboard of a/c 1.

You can try `git grep FeedbackMessageSummary` to check where this factory is used.